### ISSUE TITLE
1.2.0 doesn't install native-image

### ISSUE THREAD

**Issue by sneakythr0ws** [micronaut-projects/micronaut-profiles#175]
Hi!
I just updated MN to 1.2.0.
`mn create-app gn --features graal-native-image` generates Dockerfile 
```FROM oracle/graalvm-ce:1.0.0-rc15 as graalvm
COPY . /home/app/gn
WORKDIR /home/app/gn
RUN native-image --no-server -cp build/libs/gn-*-all.jar

FROM frolvlad/alpine-glibc
EXPOSE 8080
COPY --from=graalvm /home/app/gn .
ENTRYPOINT ["./gn"]
```

with `1.0.0-rc15` and without `RUN gu install native-image`
`./docker-build.sh` fails(

**Comment by sneakythr0ws** [comment]
I don't see this commit in 1.2.0
https://github.com/micronaut-projects/micronaut-profiles/commit/2e74f47fbc3d22847b1bd16e5c0023e9d1e3da98#diff-a0a4a0db7214cca4f30f9a84643ee993

**Comment by ilopmar** [comment]
Thanks for the report. As you discovered that's the issue. The commit will be included in Micronaut 1.2.1.
